front/app/utils/FormattedCurrency/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
Made this little helper component to replace FormattedNumber with style="currency" prop everywhere. Also had to do this to easily deal with our two new custom currencies.